src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.data.person;
+
+public class Block {
+	private String block; 
+	public Block (String block){
+		this.block=block;
```

Comment:
Operators should be surrounded by a space character.
<br>e.g. 
<br>`a = (b + c) * d;`
<br>`this.foo = foo;`